lib/chat/chat_screen.dart

```diff
@@ -1,23 +1,19 @@
-import 'dart:async';
-import 'dart:io';
-
-import 'package:file_picker/file_picker.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_chat/chat/video_view.dart';
 import 'package:flutter_chat/generated/l10n.dart';
 import 'package:flutter_chat/repository/chat_repository.dart';
+import 'package:flutter_chat/resources/assets.dart';
 import 'package:flutter_chat/util/firebase_const.dart';
 import 'package:flutter_chat/util/hero_tags.dart';
 import 'package:flutter_linkify/flutter_linkify.dart';
 import 'package:fluttertoast/fluttertoast.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:intl/intl.dart';
-import 'package:location/location.dart' hide PermissionStatus;
-import 'package:permission_handler/permission_handler.dart';
 import 'package:provider/provider.dart';
 import 'package:webview_flutter/webview_flutter.dart';
 
 import 'chat_arg.dart';
```

Comment:
It's better to use an absolute path everywhere or a relative path everywhere, to avoid hidden errors